lib/ansible/modules/network/nxos/nxos_bgp_neighbor.py

```diff
@@ -327,8 +327,11 @@ def state_present(module, existing, proposed, candidate):
             commands.append('no {0}'.format(key))
         elif value == 'default':
             if existing_commands.get(key):
-                existing_value = existing_commands.get(key)
-                commands.append('no {0} {1}'.format(key, existing_value))
+                if key == 'password':
+                    commands.append("no password")
```

Comment:
Do we have an integration test to verify this?

Reply:
@mikewiebe none of the tasks in integration tests for this target actually assert the commands sent. However, I noticed this issue when https://github.com/ansible/ansible/blob/devel/test/integration/targets/nxos_bgp_neighbor/tests/common/sanity.yaml#L221 failed locally.

Reply:
So we hit this in one of our periodic jobs too. [This](https://object-storage-ca-ymq-1.vexxhost.net/v1/a0b4156a37f9453eb4ec7db5422272df/ansible_14/63014/4fd9e871999fc13b776696b05eee0c2279fb4f85/third-party-check/ansible-test-network-integration-nxos-cli-python36/35942ee/controller/ara-report/result/e304c457-2280-4a1c-bc78-d6c38d3b9a85/) is the link.

Reply:
I ran this PR against multiple platforms and image versions and the results look good.